autogpt_platform/backend/backend/copilot/db_session_listing_test.py

```
@@ -298,3 +304,210 @@ async def test_title_search_matches_literally_and_skips_dream_sessions(
+
+
+def test_pending_question_projection_columns_exist_in_schema():
+    """Guard the hand-maintained projection in the Home pending-question query.
+
+    ``_PENDING_QUESTION_SESSION_COLUMNS`` mirrors the ChatSession schema by
+    hand, so a ``schema.prisma`` rename — or a new column with no default on
+    the generated ``PrismaChatSession`` model, which the raw query hydrates —
+    would otherwise only surface as a runtime query error on Home.
+    """
+    schema = (Path(__file__).parents[2] / "schema.prisma").read_text()
+    model = re.search(r"^model ChatSession \{(.*?)^\}", schema, re.S | re.M)
```

Ruff (0.16.1)

[error] Use of regular expression alias `re.S`

Replace with `re.DOTALL`

(FURB167)